### ISSUE TITLE
site issue

### ISSUE THREAD

**Issue by StingRay86** [WWBN/AVideo#1812]
I have some site issues.
1)The main video in YouTube first page style, is there a way to set it to not be random?
2) In the subscription page, the name of the channels subscribed to are not linked to the actual channels
3) is it possible to hide the number of videos in each category in the left navigation menu? 
Thanks

**Comment by DanielnetoDotCom** [comment]
1) Send an update
2) I don't understand what do you mean, please explain with some screenshots
3) Not yet

**Comment by StingRay86** [comment]
1) Thanks I pulled the changes, I'm on the Youtube style though. I switched to gallery anyway just to try it out but it's still loading it randomly. Is there a setting somewhere I need to set?

2) ![sub](https://user-images.githubusercontent.com/52117243/60388365-64404080-9a7e-11e9-8d67-b293afc3c221.png)
![youtube](https://user-images.githubusercontent.com/52117243/60388384-b08b8080-9a7e-11e9-88f2-8b6c480173b1.png)

3) I managed this with CSS
